src/startup/client.jsx

```diff
@@ -1,6 +1,7 @@
 import { ApolloClient, InMemoryCache } from '@apollo/client'
+import { settings } from 'settings'
 
 export const client = new ApolloClient({
-  uri: 'https://hermes.api.dev.k8s.nerds.sh/',
+  uri: settings.clientUrl,
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/128188220-371ab1ee-cf6b-474e-a09e-6d9f4c210b41.png)
Seems to me that `settings` is a function. Should we then... call it first?

Reply:
Done